### PR TITLE
Minor SEO fixes

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
## Description

Based on the Google Search Console I found some things we can do better, these are fixes for 2 minor issues.

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `flowforge/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `flowforge/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by joepavitt** [comment]
Curious, what's the issue with /careers? Because it just redirects?

File: .eleventy.js [hunk 1]
```diff
@@ -61,9 +62,12 @@ module.exports = function(eleventyConfig) {
         return heroGen(""+id)
     })
 
+    eleventyConfig.addFilter("toAbsoluteUrl", function(url) {
+        return new URL(url, site.baseURL).href;
+    })
+
```
Comment (joepavitt) [inline .eleventy.js:68]: Where is this filter used?

**Comment by ZJvandeWeg** [comment]
@joepavitt Indeed, that's the issue. Seems google likes pages that contain content

File: .eleventy.js (hunk 1 above)
Comment (ZJvandeWeg) [reply .eleventy.js:68]: https://github.com/flowforge/website/pull/370/files#diff-2c3c931c99a0419e176c109e0045c4080281011ec8beac012b5c1462b064abe5R6

Comment (ZJvandeWeg) [reply .eleventy.js:68]: Added this to more places to DRY our code.

**Review by joepavitt — APPROVED** [review]

**Pull request merged**